### PR TITLE
CSV importer

### PR DESCRIPTION
**Pull request by berggren**
Import timeline from a CSV file.

Note: There are some duplicated boiler plate in there, but this is due to how Flask is handling application context etc. Let me know if you want more details. 

Ref issue: #91 
Reviewer: @liamjm

### CODE REVIEW

File: tsctl [hunk 1]
```diff
@@ -186,80 +269,79 @@ class CreateTimelineFromJson(Command):
...
             }
         ]
 
         Args:
-            timeline_name: Name for the new timeline.
-            index_name: Name for the index in Elasticsearch.
-            filename: Path to JSON file.
-            event_type: Type of event, i.e. doc_type in Elasticsearch.
-            flush_interval: Number of events to queue up before bulk insert.
+            timeline_name: The name if the timeline in Timesketch
```
Comment (liamjm) [inline tsctl]: s/if/of/

File: tsctl [hunk 2]
```diff
@@ -186,80 +269,79 @@ class CreateTimelineFromJson(Command):
...
+            index_name: Name of the index in Elasticsearch
+            file_path: PAth to the file to process
```
Comment (liamjm) [inline tsctl]: s/PAth/Path/

**Comment by liamjm** [comment]
LGTM, couple of nits.

File: tsctl (hunk 1 above)
Comment (berggren) [inline tsctl]: Fixed.

File: tsctl (hunk 2 above)
Comment (berggren) [inline tsctl]: Fixed.

**Pull request merged**